### PR TITLE
Update user stories & NFRs

### PR DESCRIPTION
**Pull request by ziiqii**

### CODE REVIEW

**Review by zhuuyicheng — APPROVED** [review]
LGTM, nicely done comrade Xu!

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -293,24 +293,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 | `* *`    | new user                | tag suppliers with the types of food they supply                                              | easily find the right supplier for specific ingredients            |
 | `* *`    | new user                | tag employees as 'full time' or 'part time'                                                   | easily find the contacts of part time employees during peak period |
-| `* *`    | clumsy user             | see all commands I entered this session                                                       | check what I did previously                                        |
-| `* *`    | clumsy user             | undo my last command                                                                          | undo an erroneous command                                          |
+| `* *`    | clumsy user             | see all commands I entered this session                                                       |                                                                    |
```
Comment (Tanzhiheng26) [inline docs/DeveloperGuide.md:307]: As a forgetful user

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -293,24 +293,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| `* *`    | clumsy user             | undo my last command                                                                          |                                                                    |
```
Comment (Tanzhiheng26) [inline docs/DeveloperGuide.md:308]: so that I can rectify an erroneous command

**Review by Tanzhiheng26 — CHANGES_REQUESTED** [review]
Just a few nits to fix. Also your PR name is update NFRs but you updated the user stories.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ziiqii) [reply docs/DeveloperGuide.md]: Fixed

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ziiqii) [reply docs/DeveloperGuide.md]: Fixed

**Review by Tanzhiheng26 — APPROVED** [review]
LGTM

**Pull request merged**